### DISCUSSION TITLE
Keep Theme integrity when switching (i.e. themes with always-dark taskbar)

### DISCUSSION THREAD

**Discussion by ddiggles** [AutoDarkMode/Windows-Auto-Night-Mode#668] [Q&A]
**Describe the enhancement or feature you'd like**

I'm on W11 22H2.  The theme I'm currently using is "[Color Explosion](https://apps.microsoft.com/store/detail/color-explosion/9MW23Z7SD3LP?hl=en-us&gl=us&rtc=1)," but I've seen this behavior on other themes.

This theme uses a dark taskbar but light file explorer.  So the taskbar is the only element that is dark, as far as I can tell.

When ADM runs and switches to dark mode, file explorer / windows now become dark (good).   When switching to light mode, everything switches to light (including taskbar).

Is there any way to work around this and keep the dark taskbar at all times?

**Describe alternatives you've considered**

**Additional context**

![image](https://user-images.githubusercontent.com/19737405/212351166-d3ec0a16-a859-49dc-8465-94c80c7ec095.png)

**Comment by Spiritreader** [comment]
Taskbar color is tied to system mode.
![grafik](https://user-images.githubusercontent.com/18438899/212483283-e8b4e299-98d6-478b-bb4f-9326f7bd0fdd.png)

If you set system to disabled, always light or always dark, the taskbar will not change.

Anything beyond that is not possible due to how it's defined in Windows.

**Reply by LeonWenbu** [reply]
Cool! I just wanted to keep the system (task bar or window title ) dark and switch apps by time, I was looking for this feature, even I planned to write a script to implement that, until I found this discussion.

Thanks.
